### ISSUE TITLE
"Lets Work Together" section

### ISSUE THREAD

**Issue by DWL3** [ckcollab/ckcollab#64]
Change title font size:
60px

Drop us a Line button: 
Font size - 35px 
Button - width 350px, padding 10px

**Comment by Logan-Ruf** [comment]
@1080p
![image](https://user-images.githubusercontent.com/16579931/92533121-a3bbfe00-f1e6-11ea-8036-f404a82faefb.png)

@720p
![image](https://user-images.githubusercontent.com/16579931/92533172-bb938200-f1e6-11ea-9f65-6f83fcdef90d.png)

**Comment by DWL3** [comment]
@720 is like EXACTLY how it should look>> we may want to shorten the actual element by some as well, there's quite a bit of dead space below, maybe also bring the astros up a tidge and the text inward if they overlap too much

**Comment by Logan-Ruf** [comment]
I made those changes, keep in mind that the astronauts bob up and down. I tried to take the screenshots towards the top of their sine wave.
@720
![image](https://user-images.githubusercontent.com/16579931/92781260-d128b900-f358-11ea-8db0-e99f24b476c5.png)

I set upper limits on the subheader text width and the height of the div.
@1080p
![image](https://user-images.githubusercontent.com/16579931/92781383-ef8eb480-f358-11ea-9c66-1ed28058139d.png)

@4k
![image](https://user-images.githubusercontent.com/16579931/92781624-249b0700-f359-11ea-993f-8d7763ee4ee7.png)
